L1Trigger/TrackFindingTracklet/plugins/L1FPGATrackProducer.cc

```diff
@@ -689,13 +689,15 @@ void L1FPGATrackProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSe
     aTrack.setStubPtConsistency(
         StubPtConsistency::getConsistency(aTrack, theTrackerGeom, tTopo, settings.bfield(), settings.nHelixPar()));
 
-    // set TTTrack word
-    aTrack.setTrackWordBits();
+    
 
     if (trackQuality_) {
       trackQualityModel_->setTrackQuality(aTrack);
     }
 
+    // set TTTrack word
+    aTrack.setTrackWordBits();
```

Comment:
Was there any reason to put this after the if statement?

Reply:
The setTrackQuality function changes the track to fill its track MVA field which is then used by the setTrackWordBits to create the track word, the other way round and the track word doesn't contain the track MVA

Reply:
OK then indeed this makes sense